.gitignore

```diff
@@ -0,0 +1,31 @@
+# Java class files
```

Comment:
That's good idea for adding .gitignore. Would be great to have a look at https://github.com/github/gitignore where we can use at least the following:
- https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore
- https://github.com/github/gitignore/blob/master/Java.gitignore